event-management-frontend/src/components/eventCreateOrEdit/ticketsSection/CategoryPage/CategoryPageSmart.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { AppState } from "../../../../store/store";
+import { connect } from "react-redux";
+import CategoryPageDumb from "./CategoryPageDumb";
+import { EventCrud } from "../../../../model/EventCrud";
+import { addEmptyCategoryCard, updateEvent, updateFormErrors } from "../../../../actions/HeaderEventCrudActions";
+import { EventFormErrors } from "../../../../model/EventFormErrors";
+
+interface Props {
```

Comment:
Am observat ca in celelalte fisiere ai utilizat "type" si nu "interface". Pune-le pe toate la fel (fie type, fie interface)